move/sources/canvas/token.move

```diff
@@ -725,7 +724,7 @@ module addr::canvas_token {
         let object_signer = object::generate_signer_for_extending(&extend_ref);
         let new_canvas_ = Canvas {
             config,
-            pixels: smart_table::new(),
+            pixels: table::new(),
```

Comment:
this might be better as smart table: please try to do some benchmarking to determine :-)

Reply:
cc @banool for your thoughts, and if you already did the research :-)

Reply:
I did actually, quite extensively, check it out: https://www.notion.so/aptoslabs/Experimenting-with-different-vectors-map-like-structures-a380147e5f564411b4a199ef58b9f65e 🤠

Reply:
Amazing :-) thank you, I think this is a perfect framework for @0xaptosj to use for table